pallets/capacity/src/lib.rs

```diff
@@ -302,31 +302,32 @@ pub mod pallet {
 		/// Staker attempted to stake to an invalid staking target.
 		InvalidTarget,
 		/// Capacity is not available for the given MSA.
-		InsufficientBalance,
+		InsufficientCapacityBalance,
```

Comment:
This one was just to make it more specific, and to distinguish from `InsufficientStakingBalance`